persistence/data_models_enums.go

```diff
@@ -78,19 +78,22 @@ func (e StateExecutionStatus) String() string {
 	}
 }
 
-type WorkerTaskType int32
+type ImmediateTaskType int32
 
 const (
-	WorkerTaskTypeWaitUntil WorkerTaskType = 1
-	WorkerTaskTypeExecute   WorkerTaskType = 2
+	ImmediateTaskTypeWaitUntil         ImmediateTaskType = 1
+	ImmediateTaskTypeExecute           ImmediateTaskType = 2
+	ImmediateTaskTypeLocalQueueMessage ImmediateTaskType = 3
```

Comment:
Maybe `ImmediateTaskTypeLocalQueueMessageProcessing` to be more clear.

Reply:
Chosed `ImmediateTaskTypeNewLocalQueueMessage`.